### ISSUE TITLE
Python 3.8 change of location breaks AGW and FEG

### ISSUE THREAD

**Issue by uri200** [magma/magma#4678]
After upgrading to 1.5 master AGW was not able to update using APT.  The reason is all the systemd processes that use python where not able to find any of the python modules.

This is happening on TeraVM boxes

After checking for the issue we saw that the path for those files has moved from
```
/usr/local/lib/python3.5/dist-packages
```
to 
```
/usr/local/lib/python3/dist-packages
```

To be able to run this again I had to add this to the path
```
PYTHONPATH="/usr/local/lib/python3:/usr/local/lib/python3/dist-packages"
```

Note before the path were properly grab by python without us having to have a `PYTHONPATH` on our var envs

Also, I tried FEG  with the newest 1.3.3  `451400931` and when I try to bring up python containers I get this 

(note i labeled that version in jfrog as `IMAGE_VERSION=v1.3.3a`

```
magma@magma-fedgw:/var/opt/magma/docker$ docker-compose up --remove-orphans -d magmad
WARNING: The LOG_DRIVER variable is not set. Defaulting to a blank string.
WARNING: The USE_GY_FOR_AUTH_ONLY variable is not set. Defaulting to a blank string.
WARNING: The GY_SUPPORTED_VENDOR_IDS variable is not set. Defaulting to a blank string.
WARNING: The GY_SERVICE_CONTEXT_ID variable is not set. Defaulting to a blank string.
WARNING: The DOCKER_USERNAME variable is not set. Defaulting to a blank string.
WARNING: The DOCKER_PASSWORD variable is not set. Defaulting to a blank string.
Removing orphan container "s8_proxy"
Starting magmad ... error

ERROR: for magmad  Cannot start service magmad: OCI runtime create failed: container_linux.go:346: starting container process caused "exec: \"python3.8\": executable file not found in $PATH": unknown

ERROR: for magmad  Cannot start service magmad: OCI runtime create failed: container_linux.go:346: starting container process caused "exec: \"python3.8\": executable file not found in $PATH": unknown
ERROR: Encountered errors while bringing up the project.
```

**Comment by uri200** [comment]
Note this also happens on FEG on those services using python


```
[magma@192.168.60.112] out: Creating eventd        ... errorservice magmad: OCI runtime create failed: container_linux.go:346: starting
[magma@192.168.60.112] out: xec: \"python3.8\": executable file not found in $PATH": unknown

[magma@192.168.60.112] out: Creating feg_hello     ... done service eventd: OCI runtime create failed: container_linux.go:346: starting

[magma@192.168.60.112] out: Creating control_proxy ... done
[magma@192.168.60.112] out:
[magma@192.168.60.112] out: ERROR: for magmad  Cannot start service magmad: OCI runtime create failed: container_linux.go:346: starting container process caused "exec: \"python3.8\": executable file not found in $PATH": unknown
[magma@192.168.60.112] out:
[magma@192.168.60.112] out: ERROR: for eventd  Cannot start service eventd: OCI runtime create failed: container_linux.go:346: starting container process caused "exec: \"python3.8\": executable file not found in $PATH": unknown
[magma@192.168.60.112] out: ERROR: Encountered errors while bringing up the project.
[magma@192.168.60.112] out:

```

**Comment by uri200** [comment]
Hey @AyliD, I think there were some PRs added to change python 3.5 to 3.8. Now we see that both AGW and FEG are complaining about the location of python and they won't install. Can you please take a look at those errors and see if any of the changes done on those PR missed something with the path?

**Comment by tmdzk** [comment]
Should be fixed with #4877. Let's wait for the build to land. \
cc @uri200 @kkahrs
